JSDemos/Demos/Toolbar/Multiline/Angular/app/app.component.ts

```diff
@@ -0,0 +1,241 @@
+import { NgModule, Component, enableProdMode } from '@angular/core';
+import { BrowserModule } from '@angular/platform-browser';
+import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
+
+import {
+  DxToolbarModule,
+  DxSelectBoxModule,
+  DxTemplateModule,
+  DxResizableModule,
+  DxDropDownButtonModule,
+  DxButtonGroupModule,
+  DxCheckBoxModule,
+  DxRadioGroupModule,
+} from 'devextreme-angular';
+
+import DataSource from 'devextreme/data/data_source';
+import notify from 'devextreme/ui/notify';
+import {
+  FontFamilies,
+  FontSize,
+  FontStyle,
+  LineHeight,
+  ListType,
+  Service,
+  TextAlign,
+  TextAlignExtended,
+} from './app.service';
+
+if (!/localhost/.test(document.location.host)) {
+  enableProdMode();
+}
+
+@Component({
+  selector: 'demo-app',
+  templateUrl: 'app/app.component.html',
+  styleUrls: ['app/app.component.css'],
+  providers: [Service],
+})
+
+export class AppComponent {
+  toolbarLineModes = [
+    {
+      text: 'Multiline mode',
+      value: true,
+    },
+    {
+      text: 'Singleline mode',
+      value: false,
+    },
+  ];
+
+  multiline: boolean;
+
+  lineHeight: number[];
+
+  fontSizes: FontSize[];
+
+  lineHeights: LineHeight[];
+
+  fontFamilies: FontFamilies[];
+
+  fontStyles: FontStyle[];
+
+  textAlignItems: TextAlign[];
+
+  textAlignItemsExtended: TextAlignExtended[];
+
+  selectedTextAlign: string[];
+
+  listTypes: ListType[];
+
+  undoButtonOptions: any;
```

Comment:
Do we have types for these component fields?
Add the 'any' type isn't a good practice

Reply:
https://github.com/DevExpress/devextreme-demos/pull/2004/commits/3dd5d635fce75dad747b4036cbac4ef61f883e96